srcpkgs/pipewire/template

```diff
@@ -63,6 +63,8 @@ post_install() {
 	vdoc "${FILESDIR}/README.voidlinux"
 	vsv pipewire
 	vsv pipewire-pulse
+	vdoc "${FILESDIR}/pipewire.desktop"
+	vdoc "${FILESDIR}/pipewire-pulse.desktop"
```

Comment:
why `vdoc` instead of `/usr/share/applications`?

Reply:
I thought it would show up in menus, but NoDisplay prevents that. I moved it to /usr/share/applications now, thanks!